src/Map/RasterTile.cs

```diff
@@ -4,40 +4,66 @@
 // </copyright>
 //-----------------------------------------------------------------------
 
-using System;
-
-namespace Mapbox.Map {
-
-
+namespace Mapbox.Map
+{
 	/// <summary>
-	///    A raster tile from the Mapbox Style API, a encoded image representing a geographic
-	///    bounding box. Usually JPEG or PNG encoded.
+	/// A raster tile from the Mapbox Style API, an encoded image representing a geographic
+	/// bounding box. Usually JPEG or PNG encoded.
 	/// </summary>
-	public class RasterTile : Tile {
-
+	/// <example>
+	/// Making a RasterTile request:
+	/// <code>
+	/// var parameters = new Tile.Parameters();
+	/// parameters.Fs = MapboxAccess.Instance;
+	/// parameters.Id = new CanonicalTileId(_zoom, _tileCoorindateX, _tileCoordinateY);
+	/// parameters.MapId = "mapbox://styles/mapbox/satellite-v9";
+	/// var rasterTile = new RasterTile();
+	/// 
+	/// // Make the request.
+	/// rasterTile.Initialize(parameters, (Action)(() =>
+	/// {
+	/// 	if (!string.IsNullOrEmpty(rasterTile.Error))
+	/// 	{
+	///			// Handle the error.
+	///		}
+	/// 
+	/// 	// Consume the <see cref="Data"/>.
```

Comment:
We saw `xref` problems before.
Updating `docfx` solved it.

In root of `mapbox-unity-sdk` delete
* file  `docfx.zip`
* folder `docfx`

and then run `./scripts/build-docs.sh` again.

Does it work now?